lib/components/narrative/save-trip-button.js

```diff
@@ -50,14 +52,20 @@ const SaveTripButton = ({
       disabled={buttonDisabled}
       style={buttonDisabled ? { pointerEvents: 'none' } : {}}
     >
-      <i className={icon} /> {buttonText}
+      <i className={icon} />
+      {' '}
```

Comment:
I think adding whitespace is a no-no when using internationalization. If this is for styling purposes to add padding or something, the styling should be changed instead.